### PR TITLE
docs(Installation): Update Guide

### PR DESCRIPTION
**Pull request by IZOBRETATEL777**
add .deb packages installation guide
feat: Add .deb release to pipeline #568, docs: update installation guide [#56](https://github.com/cpeditor/cpeditor.github.io/pull/56)

## Type of changes

- [ ] Bug fix (changes which fix an issue)
- [ ] New feature (changes which add functionality)
- [x] Documentation (changes which modify the documentation only)
- [ ] Style change (changes which do not affect the meaning of the code: code formatting, etc.)
- [ ] Refactor (changes which affect the meaning of the code but neither fix a bug nor add a feature)
- [ ] Performance improve (changes which improve performance)
- [ ] Test (changes which add tests)
- [ ] Build (Changes that affect the build system or external dependencies)
- [ ] CI (changes to CI configuration files and scripts)
- [ ] Chore (changes which do not belong to any type above)
- [ ] Revert (revert previous changes)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/cpeditor/cpeditor/blob/master/CONTRIBUTING.md) document.
- [x] I have tested these changes locally, and this fixes the bug/the new feature behaves as the expectation.
- [x] The settings file in the old version can be used in the new version after this change.
- [x] These changes only fix a single bug/introduces a single feature. (Otherwise, open multiple Pull Requests instead, unless these bugs/features are closely related.)
- [x] The commit messages are clear and detailed. (Otherwise, use `git reset` and commit again, or use `git rebase -i` and `git commit --amend` to modify the commit messages.)
- [x] These changes don't remove an existing feature. (Otherwise, add an option to disable this feature instead, unless it's necessary to remove this feature.)
- [x] If there are changes of the text displayed in the UI, I have wrapped them in `tr()` or `QCoreApplication::translate()`.
- [ ] I have documented these changes in [CHANGELOG.md](https://github.com/cpeditor/cpeditor/blob/master/doc/CHANGELOG.md), or these changes are not notable.

### CODE REVIEW

File: doc/INSTALL.md [hunk 1]
```diff
@@ -28,6 +28,14 @@ Download the `setup.exe` in the assets and install it. You may also need to inst
 
 Download the `AppImage` in the assets, add execute permission to it (`chmod +x <AppImagePath>`), then run it.
 
+On Debian-based Linux distributions (Ubuntu, Linux Mint and etc.) you can download `.deb` package in the assets and run the command in terminal:
```
Comment (coder3101) [inline doc/INSTALL.md:31]: `and etc` doesn't seems ok? Keep it just `etc` (no full stop)

**Review by coder3101 — CHANGES_REQUESTED** [review]

**Comment by coder3101** [comment]
@ouuan can you translate this to Chinese Readme.

**Comment by StudyingFather** [comment]
> @ouuan can you translate this to Chinese Readme.

Maybe I can help to translate it.

**Comment by coder3101** [comment]
@StudyingFather Sure! For sometime ouuan will not be very active and so we want a person to help us with chinese translation.

**Comment by coder3101** [comment]
@StudyingFather I will mention you whenever, I want a translation update from your side. In order to translate, you will need Qt Linguistic Application (Source code translations) for Readme and docs, you can edit them as you wish.

**Comment by ouuan** [comment]
I think it's better to start translating the website than to maintain two documents :thinking:

**Comment by coder3101** [comment]
> I think it's better to start translating the website than to maintain two documents :thinking:

Yes, this PR is last time we will translate markdowns. It will only help while translating website docs. (Stuff can be copy pasted)

**Review by coder3101 — APPROVED** [review]

**Comment by ouuan** [comment]
We are removing the documentation in the docs directory, so cpeditor/cpeditor.github.io#56 takes over this. The translations can be moved to the website later.

**Pull request closed without merging**